orbitx/flight_gui.py

```diff
@@ -32,73 +34,37 @@
 class FlightGui:
 
     def __init__(self, physical_state_to_draw: protos.PhysicalState, texture_path: str =None, no_intro: bool =False )-> None: 
-        ctrl_c_handler: function = signal.getsignal(signal.SIGINT)
         # Note that this might actually start an HTTP server!
-        #import vpython
```

Comment:
anything that takes out this line is good, it was always a gross workaround to begin with. But see my comments in `Vpython.py`

Reply:
without import vpython in module scope, there is no way to add type annotation for function returning VPython object. Because static type check produces error.